src/app/pages/data-protection/cloud-backup/cloud-backup-form/cloud-backup-form.component.html

```diff
@@ -0,0 +1,94 @@
+<ix-modal-header2 [requiredRoles]="requiredRoles" [title]="title" [loading]="isLoading"></ix-modal-header2>
+<mat-card>
+  <mat-card-content>
+    <form class="ix-form-container" [formGroup]="form" (submit)="onSubmit()">
+      <div class="fieldsets">
+        <ix-fieldset
+          class="fieldset"
+          [title]="'Local' | translate"
+        >
+          <ix-explorer
+            formControlName="path"
+            [label]="'Source Path' | translate"
+            [required]="true"
+            [multiple]="true"
+            [nodeProvider]="fileNodeProvider"
+          ></ix-explorer>
+        </ix-fieldset>
+        <ix-fieldset
+          class="fieldset"
+          [title]="'Remote' | translate"
+        >
+          <ix-cloud-credentials-select
+            formControlName="credentials"
+            [label]="'Credentials' | translate"
+            [required]="true"
+          ></ix-cloud-credentials-select>
+          <ix-select
+            *ngIf="form.controls.bucket.enabled"
+            formControlName="bucket"
+            [label]="'Bucket' | translate"
+            [options]="bucketOptions$"
+            [required]="true"
+          ></ix-select>
+          <ix-input
+            *ngIf="form.controls.bucket_input.enabled"
+            formControlName="bucket_input"
+            [label]="'New Backet Name' | translate"
+          ></ix-input>
+          <ix-explorer
+            *ngIf="form.controls.folder.enabled"
+            formControlName="folder"
+            root="/"
+            [label]="'Folder' | translate"
+            [nodeProvider]="bucketNodeProvider"
+            [required]="true"
+          ></ix-explorer>
+        </ix-fieldset>
+        <ix-fieldset
+          class="fieldset"
+          [title]="'Task Settings' | translate"
+        >
+          <ix-input
+            formControlName="description"
+            [label]="'Description' | translate"
+          ></ix-input>
+          <ix-input
+            formControlName="keep_last"
+            type="number"
+            [label]="'Keep Last' | translate"
+            [required]="true"
+          ></ix-input>
+          <ix-input
+            formControlName="password"
+            type="password"
+            [label]="'Password' | translate"
+            [required]="true"
+          ></ix-input>
+          <ix-scheduler
+            formControlName="schedule"
+            [required]="true"
+            [label]="'Schedule' | translate"
+          ></ix-scheduler>
+        </ix-fieldset>
+      </div>
+
+      <ix-form-actions>
+        <button mat-button type="button" matDialogClose ixTest="cancel">
```

Comment:
I think we don't need this button, it's not working for me.
We don't have Cancel buttons on such similar forms.